test/trigger-circleci-workflows.test.ts

```diff
@@ -8,9 +8,18 @@ nock.disableNetConnect();
 
 const EXAMPLE_CONFIG = `
 labels_to_circle_params:
-  ci/binaries: run_binaries_tests
-  ci/bleh: run_bleh_tests
-  ci/foo: run_foo_tests
+  ci/binaries:
+    parameter: run_binaries_tests
+    default_true_on:
+      branches:
+        - nightly
+        - ci-all/.*
+      tags:
+        - v[0-9]+(\.[0-9]+)*-rc[0-9]+
+  ci/bleh:
+    parameter: run_bleh_tests
+  ci/foo:
+    parameter: run_foo_tests
 `;
```

Comment:
There ought to be some docs for this configuration format, probably in the README